### ISSUE TITLE
Properties of const objects are read-only

### ISSUE THREAD

**Issue by razh** [6to5/6to5#770]
After upgrading to 3.6.0:

```js
const x = {};
x.key = "value";
```

Results in:

```js
repl: Line 2: "x" is read-only
  1  | const x = {};
> 2  | x.key = 'value';
     | ^
```

**Comment by sebmck** [comment]
Gah, this is a regression, sorry about that. I'll push out a new version tonight.

**Comment by sebmck** [comment]
Fixed as of 3.6.1, thanks!
